main.py

```diff
@@ -1,574 +1,176 @@
-                        img = gamma_correction(img, .4)
-
-                        for face_landmarks in self.multi_face_landmarks:
-
-                                # INDICIES: https://github.com/tensorflow/tfjs-models/blob/838611c02f51159afdd77469ce67f0e26b7bbb23/face-landmarks-detection/src/mediapipe-facemesh/keypoints.ts
-                                # https://github.com/google/mediapipe/blob/master/mediapipe/python/solutions/face_mesh_connections.py
-
-                                self.mpDraw.draw_landmarks(
-                                    image=tmp,
-                                    landmark_list=face_landmarks,
-                                    connections=self.mpFace.FACEMESH_RIGHT_EYE,
-                                    landmark_drawing_spec=None,
-                                    connection_drawing_spec=self.mpDraw.DrawingSpec(color=(255, 150, 255), thickness=1, circle_radius=1))
-                                    # connection_drawing_spec=self.mpDrawStyles
-                                    # .get_default_face_mesh_contours_style())
-
-                                self.mpDraw.draw_landmarks(
-                                    image=tmp,
-                                    landmark_list=face_landmarks,
-                                    connections=self.mpFace.FACEMESH_LEFT_EYE,
-                                    landmark_drawing_spec=None,
-                                    connection_drawing_spec=self.mpDraw.DrawingSpec(color=(255, 255, 0), thickness=1, circle_radius=1))
-                                    # connection_drawing_spec=self.mpDrawStyles
-                                    # .get_default_face_mesh_contours_style())
-
-                        img[y:y+h, x:x+w] = tmp
-
-                        img = cv2.resize(img, (960, 540))
-                        cv2.imshow('baby', maintain_aspect_ratio_resize(img, width=self.frame_dim[0], height=self.frame_dim[1]))
-
-                        if cv2.waitKey(1) & 0xFF == ord('q'):
-                            break
-                    except Exception as e:
-                        print("Something went wrong: ", e)
-
-
-####################################
-# TODO: move out of this file, break it up
-
-print('Initializing...')
-sleepy_baby = SleepyBaby()
-print('\nInitialization complete.')
-
-
-# Below http server is used for the web app to request latest sleep data
-class CORSRequestHandler(SimpleHTTPRequestHandler):
-    def end_headers(self):
-        self.send_header('Access-Control-Allow-Origin', '*')
-        self.send_header('Access-Control-Allow-Methods', 'GET')
-        self.send_header('Cache-Control', 'no-store, no-cache, must-revalidate')
-        return super(CORSRequestHandler, self).end_headers()
+#Load configuration from .env file
+config = dotenv_values()
 
-def start_server():
-    httpd = HTTPServer(('0.0.0.0', 8000), CORSRequestHandler)
-    httpd.serve_forever()
+#Config command ling
+parser = argparse.ArgumentParser(
```

Comment:
Mind adding `argparse` to the `requirements.txt`? And adding it as an import in this file?

Reply:
Ah interesting, never heard of it. Looks cool, sounds good. Thanks Salvo

Reply:
with latest commits, I have solved this issue, now requirements.txt has been aligned with used libraries.